guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -1,28 +1,272 @@
-[[Importing_Red_Hat_Content]]
-== Importing Red{nbsp}Hat Content
+[[Importing_Content]]
+== Importing Content
 
-This section describes how to use products and repositories in {Project}, and to create synchronization plans to ensure your {Project} content remains up to date with the content on the Red{nbsp}Hat Content delivery network (CDN).
+This chapter outlines how you can import {customcontent} of different types to {Project}.
```

Comment:
In Satellite docs, this will render as custom content of different types...

Reply:
Maybe 'how you can import different types of {customcontent}' sounds better?